include/aspect/material_model/rheology/elasticity.h

```diff
@@ -0,0 +1,159 @@
+/*
+  Copyright (C) 2019 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef _aspect_material_model_rheology_elasticity_h
+#define _aspect_material_model_rheology_elasticity_h
+
+#include <aspect/global.h>
+#include <aspect/material_model/interface.h>
+#include <aspect/simulator_access.h>
+
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    using namespace dealii;
+
+    namespace Rheology
+    {
+      /**
+       * Additional output fields for the elastic shear modulus to be added to
+       * the MaterialModel::MaterialModelOutputs structure and filled in the
+       * MaterialModel::Interface::evaluate() function.
+       */
+      template <int dim>
+      class ElasticAdditionalOutputs : public NamedAdditionalMaterialOutputs<dim>
```

Comment:
Is putting this into the Rheology namespace intentional?

Reply:
Well I thought elastic outputs are something that's really part of the rheology model, so I just put it into that namespace. What would be a reason for moving it out of the namespace?

Reply:
The reason I mention this is that you are moving this class from the MaterialModel namespace. I am not sure if others are using this class, but you would break those plugins.

Reply:
I checked, an the functionality was added more than a year ago (#1593). I'll move this out of the namespace.